src/ibex_bluesky_core/devices/dae/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""Utilities for the DAE device - mostly XML helpers."""
+
+from enum import Enum
+from typing import Any, Dict, List
+from xml.etree.ElementTree import Element
+
+
+def convert_xml_to_names_and_values(xml: Element) -> Dict[str, str]:
+    """Convert an XML element's children to a dict containing <Name>.text:<Val>.text."""
+    names_and_values = dict()
+    elements = get_all_elements_in_xml_with_child_called_name(xml)
+    for element in elements:
+        name, value = _get_names_and_values(element)
+        if name is not None:
+            names_and_values[name] = value
+    return names_and_values
+
+
+def get_all_elements_in_xml_with_child_called_name(xml: Element) -> List[Element]:
+    """Find all elements with a "name" element, but ignore the first one as it's the root."""
+    elements = xml.findall(".//Name/..")[1:]
```

Comment:
More robust to act on the 'cluster' element rather than 'one level down'.  e.g. `xml.find("Cluster").findall(".//Name/..")`